CONTRIBUTING.md

```diff
@@ -1,62 +1,152 @@
-  - Ensure the icon is horizontally and vertically centered.
-  - Ensure that all paths and strokes have been converted to fills.
-  - Ensure your SVG is monochromatic. Remove all fill colors so that icons default to black.
-  - Minify your SVG after exporting it. We recommend using [SVGO](https://github.com/svg/svgo) or [SVGOMG](https://jakearchibald.github.io/svgomg/) to automate this process. **Do not go below a precision of 3**.
+## Table of contents
 
-## JSON data for simpleicons.org
+* [How To Request An Icon](#how-to-request-an-icon)
+* [How To Add An Icon](#how-to-add-an-icon)
+* [How To Label Issues](#how-to-label-issues)
+* [How To Build Locally](#how-to-build-locally)
 
-In addition to following the guidelines for SVGs, list new icons in the `_data/simple-icons.json` file. Each icon in the array has three required values:
+## How To Request An Icon
 
-  - The `title` of the new SVG.
-  - A `hex` color value that matches the brand or logo's main accent color. (Without the `#` pound symbol.)
-  - The `source` URL of the logo being used.
+We welcome icon requests. However, before you submit a new issue please make sure the icon:
 
-Here is the object for the GitHub logo as an example:
+* Is a _popular_ brand.
+* Has not already been requested.
+* Can be represented in a monochromatic format.
 
-```json
-{
-  "title": "GitHub",
-  "hex": "181717",
-  "source": "https://github.com/logos"
-},
-```
+When submitting a new icon request, please submit the issue with the following information:
+
+* **Title:** Include the brand name. Example: `Request: GitHub Icon`
+* **Body:** Include links to official sources for the brand's icon and official colors (e.g. media kits, brand guidelines, SVG files etc.)
+
+## How To Add An Icon
+
+### 1. Identify Official Logos and Colors
+
+Most of the icons and brand colors on SimpleIcons have been derived from official sources. Using official sources helps ensure that the icons and colors in SimpleIcons accurately match the brand they represent. Thankfully, this is usually a simple process as organizations often provide brand guides and high quality versions of their logo for download.
+
+Official high quality brand logos and brand colors can usually be found in the following locations:
+
+1. About pages, Press pages, Media Kits, and Brand Guidelines.
+1. Website headers
+1. Favicons
+1. Wikimedia (which should provide a source)
+
+Working with an SVG version of the logo is best. In the absence of an SVG version, other vector filetypes may work as well such as EPS. In the absence of vector logos, a vector can be created from a high quality rasterized image, however this is much more labor intensive.
+
+For color, the brand's primary web color should be used. Simple Icons stores brand colors in the standard 6 character hexadecimal format.
```

Comment:
The hex codes are often also provided in the About pages/Press pages/Media Kits/Brand Guidelines, so I think we should mention that as well

Reply:
That should be covered by the statement "Official high quality brand logos and brand colors can usually be found in the following locations." seen a few lines up.